.github/workflows/ci-style.yml

```diff
@@ -28,11 +23,11 @@ jobs:
         sudo apt install libkrb5-dev ruby ruby-dev
 
     - name: Install python dependencies
-      run: |
-        pip install --upgrade pip
-        pip install -r requirements/requirements-py-3.10.txt
-        pip install -e .[pre-commit]
-        pip freeze
+      uses: ./.github/actions/install-aiida-core
+      with:
+        python-version: '3.10'
+        extras: '[pre-commit]'
+        from-requirements: 'true'
```

Comment:
I realize that this was already the case, but why are we installing from requirements and then normally with the extra? I presume because we cannot specify the extra when installing from requirements. But then why not simply get rid of the requirements install. I don't think we need it for the pre-commit step

Reply:
I thought so as well, but if I don't install from requirements first the pre-commit will actually fail, I think because it is missing pytest library. I'll open a separate issue for that.

Reply:
Opened #6366